README.md

```diff
@@ -146,35 +147,35 @@ for grabber in grabbers.values():
 ```
 ### Configurations
 The table below shows all available configurations and the cameras to which they apply.
-| Configuration Name         | Example         | Generic USB     | RTSP      | Basler    | Realsense |
-|----------------------------|-----------------|------------|-----------|-----------|-----------|
-| name                       | On Robot Arm    | optional   | optional  | optional  | optional  |
-| input_type                 | generic_usb    | required   | required  | required  | required  |
-| id.serial_number           | 23458234       | optional   | -         | optional  | optional  |
-| id.rtsp_url                | rtsp://…        | -          | required  | -         | -         |
-| options.resolution.height  | 480            | optional   | -         | -         | optional  |
-| options.resolution.width   | 640            | optional   | -         | -         | optional  |
-| options.zoom.digital       | 1.3            | optional   | optional  | optional  | optional  |
-| options.crop.pixels.top    | 100            | optional   | optional  | optional  | optional  |
-| options.crop.pixels.bottom | 400            | optional   | optional  | optional  | optional  |
-| options.crop.pixels.left   | 100            | optional   | optional  | optional  | optional  |
-| options.crop.pixels.right  | 400            | optional   | optional  | optional  | optional  |
-| options.crop.relative.top  | 0.1            | optional   | optional  | optional  | optional  |
-| options.crop.relative.bottom | 0.9          | optional   | optional  | optional  | optional  |
-| options.crop.relative.left | 0.1            | optional   | optional  | optional  | optional  |
-| options.crop.relative.right | 0.9            | optional   | optional  | optional  | optional  |
-| options.depth.side_by_side | 1              | -          | -         | -         | optional  |
-| options.num_90_deg_rotations | 2              | optional          | optional         | optional         | optional  |
-| options.keep_connection_open | True              | -          | optional         | -         | -  |
-| options.max_fps | 30              | -          | optional         | -         | -  |
+| Configuration Name         | Example         | Generic USB     | RTSP      | Basler    | Realsense | Raspberry Pi CSI2 |
+|----------------------------|-----------------|------------|-----------|-----------|-----------|-----------|
+| name                       | On Robot Arm    | optional   | optional  | optional  | optional  | optional  |
+| input_type                 | generic_usb    | required   | required  | required  | required  | required  |
+| id.serial_number           | 23458234       | optional   | -         | optional  | optional  | optional  |
```

Comment:
Do Raspberry Pi CSI2 cameras actually have a serial number that we can recognize? When you create a grabber and then print its config, do you see a serial number? Other grabber types such as Generic USB will discover the serial number (linux only). Also, you can provide a serial number to disambiguate between different cameras. Since this new grabber type only supports one grabber, providing a serial number wouldn't be necessary (except maybe to differentiate between a USB cam that might be confused for a Raspberry Pi CSI2. In any case, if we cannot discover or use the serial numbers for these cameras, maybe this chart should say `-` for id.serial_number for Raspberry Pi CSI2.